### DISCUSSION TITLE
low bridge battery notification G7

### DISCUSSION THREAD

**Discussion by dbfreud** [NightscoutFoundation/xDrip#3095] [Q&A]
I'm getting notifications of low bridge battery, it's now at 9% but I have nearly 5 days left on my G7 transmitter.  This is my first week using xdrip

**Comment by Navid200** [comment]
I am surprised.  
G7 has no dependence on a bridge.

Would you please show a screenshot of the notification?

**Comment by dbfreud** [comment]
Navid.

Thanks for your reply. The notification only occurs periodically, I believe
the first notification was at 28 percent. Yesterday it notified me at 10
then again at 9 but I haven't seen anything today. If/when it appears again
I'll let you know.

David

**Reply by Navid200** [reply]
Most notifications have associated logs.
Please go to the logs and scroll up and down to go to the time you received the last notification.  Do you see an associated log?
Can you show it?
https://navid200.github.io/xDrip/docs/Logs.html

**Comment by dbfreud** [comment]
Searching for a record from yesterday, but looks like I never shut off the
detailed log command, so there's a crazy number of logs.  Could that be
drawing the transmitter battery?

**Comment by dbfreud** [comment]
here is the log searching battery but I don't see record of the low bridge
battery message

**Reply by Navid200** [reply]
This post explains how to attach a screenshot.
Replying to an email notification is not the way to use these discussions.
We want to help you.  But, you need to use the tools correctly.

**Comment by dbfreud** [comment]
I don't understand what you want me to do that I'm not doing.  I attached a
screen shot. Your signature states that I may reply to your email
directly.  I'm glad to follow your protocols, please advise

**Reply by Navid200** [reply]
I'm sorry I forgot to add a link.  This the link:
https://github.com/NightscoutFoundation/xDrip/discussions/1900

It asks you to not respond to emails in order to post here.

If you log into GitHub and view this discussion, you can see that there is not a single image attached to any of your posts.
Therefore, I cannot see any of the screenshots you think you have attached.

**Reply by dbfreud** [reply]
![Screenshot_20230923-162627_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/145223961/87f3905a-e44b-4b6b-a9ab-2832a7cb775d)

**Reply by dbfreud** [reply]
I'm sending the same screenshot through GitHub. Please confirm you can now view?

**Reply by Navid200** [reply]
You can see what everyone, including me, can see if/when you are logged into GitHub and view these forums.  

The logs you are showing are not related to a bridge.  Those are all the about the G7 device battery.

I'm not sure why you are seeing a notification related to bridge battery.
I have to say you can ignore it.
But, why you see the notification, I am not sure.
I need to see it.

**Reply by dbfreud** [reply]
I'm aware that those logs don't relate to the bridge.; the reason I sent was show that there is no log referencing the low bridge message.  Asking again: do you know whether  leaving active the command to generate detailed logs would drain the transmitter battery?

**Reply by Navid200** [reply]
No, the Dexcom transmitter behavior is not affected by enabling xDrip detail logs.  It only affects xDrip's behavior.

**Comment by jonprogers** [comment]
@dbfreud This may be a totally way off comment, and I know this may sound silly, but are you sure this is an xDrip warning?

Let me explain, years ago when I was using G4 with the wixel bridge (hand built bit of tech used before dexcom adopted bluetooth) to allow xDrip to collect the data I was trying to find a way to monitor the battery in the wixel bridge. I thought the bridge battery that nightscout displayed was my answer although when I tested it I found that that was reporting the phone battery - correctly as that was the bridge to nightscout, rather than the wixel battery - the bridge between G4 and xDrip.

As I think there is no bridge device between G7 and xDrip (at least there isn't for my G6 and hasn't been since G4) and there is no log message in xDrip to say a notification was raised (I've always been able to find xDrip log entries for xDrip notifications when I have looked for them) is there a possibility that the notification you are seeing is from something, somewhere else in your setup.

Please feel free to ignore me if you feel I'm stating the obvious and/or none of this is relevant to your situation. It will be interesting if/when you can grab a screenshot of the message...

**Comment by jamorham** [comment]
If we can get a screenshot of the wording of the notification we can track down what is causing it. I'm not sure if any unusual battery readings on G7 could cause a notification but if they do we should probably exclude them as there really is no point alerting the user as the sensor either works or it doesn't.

**Reply by dbfreud** [reply]
thanks to you both  The notification referencing another device seems a plausible explanation, which would be very good news as it means my transmitter battery is not about to die. I've never seen a notification with the term "bridge" before installing xdrop. I'll be sure to capture if/when it appears thanks

**Reply by dbfreud** [reply]
the notification appeared again. here's a screenshot

![Screenshot_20231001-074632_Phone.jpg](https://github.com/NightscoutFoundation/xDrip/assets/145223961/82231eb4-5e0a-45be-bebb-50a48d202e60)

**Comment by jamorham** [comment]
@dbfreud please go to `Home Screen` -> `top left burger menu` -> `Settings` -> `Alarms and Alerts` -> `Extra Alerts (xDrip+)` -> `Collector battery alerts` -> `Switch to off` 

This should resolve your problem while we investigate it further. Thanks.

**Comment by jamorham** [comment]
I don't see a code path relating to the collector used with G7 and bridge battery alerts. Could this be something to do with your wear device? Could that have any old collector settings still active or could the alert be happening when your wear device gets low battery?

**Comment by dbfreud** [comment]
the notification could have been initiated by low battery on the wear
device but remained after it was charged. I'm fine to ignore this now that
you've determined it wasn't notifying me that the transmitter battery is
low.  Thanks.
